addon/lib/field.js

```diff
@@ -330,6 +331,37 @@ export default Base.extend({
     return this.question.isMultipleChoice ? selected : selected[0];
   }),
 
+  /**
+   * The field's JEXL context.
+   *
+   * Properties:
+   * - `form`: Legacy property pointing to the root form.
+   * - `info.form`: The form this question is attached to.
+   * - `info.parent.form`: The parent form if applicable.
+   * - `info.root.form`: The new property for the root form.
+   */
+  jexlContext: computed(
+    "document.jexlContext.{form,info.root.form}",
+    "fieldset.{form.slug,field.fieldset.form.slug}",
+    function() {
+      const context = copy(this.document.jexlContext);
+
+      // This condition is needed to allow for partial states during testing.
+      if (this.get("fieldset")) {
```

Comment:
```suggestion
      if (this.fieldset) {
```

Reply:
Also, if this is only necessary for testing then maybe the tests should be adjusted to match reality

Reply:
I'll look into adjusting the tests.

Reply:
Removed the condition.